src/python_testing/matter_testing_infrastructure/matter/testing/matter_testing.py

```
@@ -1841,6 +1841,11 @@ def is_pics_sdk_ci_only(self) -> bool:
         """Checks if the 'PICS_SDK_CI_ONLY' PICS flag is enabled."""
         return self.check_pics('PICS_SDK_CI_ONLY')
 
+    @property
+    def data_model(self) -> XmlDataModel:
```

Flake8 (7.3.0)

[error] undefined name 'XmlDataModel'

(F821)